data/sql/updates/pending_db_world/rev_1687759233155225800.sql

```diff
@@ -0,0 +1,26 @@
+-- Fix SAI Yenniku --
+DELETE FROM `creature_text` WHERE (`CreatureID` = 2530 AND `GroupID` = 0);
+INSERT INTO `creature_text` (`CreatureID`, `GroupID`, `ID`, `Text`, `Type`, `Language`, `Probability`, `Emote`, `Duration`, `Sound`, `BroadcastTextId`, `TextRange`, `comment`) VALUES
+(2530, 0, 0, '%s is struck dumb by the Soul Gem!', 16, 0, 0, 0, 1500, 0, 681, 0, ' ');
```

Comment:
Missing a comment, probability (default at 100).
Doesn't need Duration, especially since it's an emote
```suggestion
(2530, 0, 0, '%s is struck dumb by the Soul Gem!', 16, 0, 100, 0, 0, 0, 681, 0, 'Yenniku');
```

Reply:
Done